src/pages/profile/Profile.jsx

```diff
@@ -50,14 +52,32 @@ export const Profile = () => {
         </Box>
       </Box>
 
-      <Button
+      {/* these comments will get removed in subsequent PR's */}
+
+      {/* <Button
         variant="outlined"
         onClick={() => setOpenEditProfileDialog(true)}
-        className={styles.btn_editProfile}
+        className={classNames(styles.btn, styles.btn_editProfile)}
       >
         Edit Profile
+      </Button> */}
+
+      <Button
+        variant="contained"
+        onClick={() => setOpenEditProfileDialog(true)}
+        className={classNames(styles.btn, styles.btn_follow)}
+      >
+        Follow
       </Button>
 
+      {/* <Button
```

Comment:
can you remove unused code

Reply:
Yeah, they will get removed in subsequent PR's